public/manifest.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "shelf-webapp",
+  "lang": "en-US",
+  "short_name": "shelf",
+  "description": "Modern Open Source Asset Management System",
+  "start_url": "/",
+  "display": "standalone",
+  "scope": "/",
+  "theme_color": "#EF6820",
```

Comment:
When I tested the PWA i can see the bar on top of my iphone is orange. I suspect this is the theme color. Can you change it to the same color as our app's background color so we see how it looks like?

Reply:
Yup will change that